scss/modules/tiles.scss

```diff
@@ -0,0 +1,72 @@
+//
+// Tiles (Datagrid decorator)
+// --------------------------------------------------
+
+.tiles-view {
+    $spacing: 20px;
+
+    margin: 0 (-$spacing) (-$spacing) 0;
+
+    .tile,
+    .add-new-tile {
+        width: 120px;
+        height: 120px;
+        display: inline-block;
+        margin: 0 $spacing $spacing 0;
+        cursor: pointer;
+        text-align: center;
+        vertical-align: middle;
+        border-radius: $borderRadiusSmall;
+    }
+
+    .tile {
+        border-radius: $borderRadiusSmall;
```

Comment:
Isn't that the same as defined in the previous selector? Why do you need a second selector for the tile solely at all? Does it look different?